decorators/ng-module/index.ts

```diff
@@ -0,0 +1,24 @@
+import importHandler from "./metadata-handlers/import";
+import daclarationHandler from "./metadata-handlers/declaration";
+import providerHandler from "./metadata-handlers/provider";
+
+export function NgModule({imports, declarations, providers, ng1RunConfig}: any) {
+    return function (target: any) {
+        let ng1ModuleIds: Array<string> = [];
+        let ng1Module = angular.module(target.name, ng1ModuleIds);
```

Comment:
Consider moving this below the first condition as we may face problems with dependencies.